HELPS/ClientApp/src/App.tsx

```diff
@@ -1,5 +1,10 @@
 import {Route} from 'react-router';
 import Layout from './components/navigation/Layout';
+<<<<<<< HEAD
```

Comment:
Looks like you have used the git command line to merge here, make sure you remove these before committing.

Alternatively, you can use VSCode to handle merging to save a bit of trouble - https://code.visualstudio.com/docs/editor/versioncontrol